src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -175,6 +175,14 @@ registerSendSequenceKeybinding(String.fromCharCode('W'.charCodeAt(0) - 64), {
 	primary: KeyMod.CtrlCmd | KeyCode.Backspace,
 	mac: { primary: KeyMod.Alt | KeyCode.Backspace }
 });
+if (platform.isWindows) {
+	// Delete word left: ctrl+h
+	// Windows cmd.exe requires ^H to delete full word left
+	registerSendSequenceKeybinding(String.fromCharCode('H'.charCodeAt(0) - 64), {
```

Comment:
64 is a magic number. It's clearer to use `String.fromCharCode('H'.charCodeAt(0) - 'A'.charCodeAt(0) + 1)`, this way the code looks like it gets the index of H in the alphabet, which is what you intended.

But actually, `^H` is just a specific notation. This expression is actually equal to `'\b'`, which is the backspace character in ASCII.

Use `'\b'` instead.

Reply:
I have been vacillating on this change quite a lot. 

Would `'\b'` not similarly be a magic string in this instance? Keeping in mind that this code [emulates the surrounding lines](https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts#L163), I wonder if this is truly an improvement -- is it common knowledge that `'\b'` is `0x08`?

An alternative would be to have a constant similar to `const ALPHABET_OFFSET = 'A'.charCodeAt(0) - 1;`

That said, this is an inconsequential change and if @Tyriar agrees I would be happy to make it.

Reply:
> String.fromCharCode('H'.charCodeAt(0) - 'A'.charCodeAt(0) + 1)

I prefer this over `\b` as it's trying to show `^H`, I'll make the change.